apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -4633,6 +4655,52 @@ sophia_crypto(_Cfg) ->
     ?assertMatchFATE({bytes, Sha256},    ResSha256),
     ?assertMatchFATE({bytes, Blake2b},   ResBlake2b),
 
+    %% Tests for bytes (which should hash as raw strings from lima)
+
+    case vm_version() >= ?VM_AEVM_SOPHIA_3 of   %% No bytes in minerva
```

Comment:
This looks like it could make use of `?skipRest` to reduce the nesting.

Reply:
There are tests below that shouldn't be skipped...